src/loader-components/_SrcCodeBadge.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './BadgeCommon.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * url to link to
+   */
+  url: PropTypes.string,
+  /**
+   * The package version.
+   */
+  version: PropTypes.string.isRequired,
+};
+
+const SrcCodeBadge = ({ url, version }) => (
```

Comment:
I don't know if it makes sense to show the version behind the code as a badge. Version would get doubled up sinc we'd still show the npm version badge.

Reply:
In my head when I logged this issue, I was expecting something like this:
```jsx
import IconDocuments from 'terra-icon/lib/icon/IconDocuments';

...
  <div className={cx('badge-container')}>
    <a className={cx('badge')} href={url}>
      <span className={cx('badge-name')}>
        {'view src'}
      </span>
     <IconDocuments /> <--- or an image of something to represent src code
    </a>
  </div>
```

Reply:
![Screen Shot 2020-03-27 at 3 15 04 PM](https://user-images.githubusercontent.com/14099737/77796681-c34ada00-703d-11ea-94af-12ecbf97d879.png)
(whatchya think of my powerpoint skills for this pic? 😋 )

Reply:
What would you think of just using a terra-hyperlink?

Reply:
I mean... I don't really love the idea but I'm not going to really fit it either. It's almost exists now. I just think it'd be nice if this was built in